_posts/2022-04-06-#24571.md

```diff
@@ -10,15 +10,49 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+* Fingerprinting allows an attacker to recognize the same node across different
```

Comment:
```suggestion
* Attackers may use fingerprinting techniques to recognize the same node across different
```

Reply:
Could also link to a wiki article since it's a broad term https://en.wikipedia.org/wiki/Device_fingerprint